conductor/resources/resources.yml

```diff
@@ -1183,64 +1183,82 @@ package_ids:
                 "2.1.15": bd0be834be4e6894bfb31e0de5548d16
                 "2.2.1": 77e9e1dbfbf273dc1a3f6d676c310d0a
 
+        "Autodesk Maya 2018":
+            package: 460411ee5c24661ed354863bcb3d7c21
```

Comment:
Is this the package id for Maya 2018 or  MayaIO 2018?  It should be for MayaIO 2018

Reply:
This is for MayaIO 2018